contrib/dracut/02zfsexpandknowledge/module-setup.sh.in

```diff
@@ -57,6 +57,14 @@ array_contains () {
 }
 
 check() {
+
+    # https://github.com/dracutdevs/dracut/pull/1711 provides a zfs_devs
+    # function to detect the physical devices backing zfs pools. If this
+    # function exists in the version of dracut this module is being called
+    # from, then it does not need to run.
+
+    type zfs_devs >/dev/null && return 1
```

Comment:
you also want `2>&1` here to swallow error spew on older dracuts (and, since you're touching it already, please drop the empty line after `check() {` for style consistency)

Reply:
Done!